### PR TITLE
27. Add component sign up, sign in, sign up note

### PR DESCRIPTION
**Pull request by thanhdo1991**
Please check and merge code

### CODE REVIEW

File: scss/components/_modal.scss [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+  &.in {
+    @include opacity(.5);
+  }
+}
+
+.modal-forms-modal-content {
+  @include breakpoint($desktop-small) {
+    margin-left: auto;
+    margin-right: auto;
+    margin-top: 150px;
```
Comment (bienvu) [inline scss/components/_modal.scss]: need change to rem

Comment (thanhdo1991) [inline scss/components/_modal.scss]: fixed

Comment (thanhdo1991) [inline scss/components/_modal.scss]: Update media query

**Comment by thanhdo1991** [comment]
Please check code

**Pull request merged**